06-building-a-program-start-to-finish/automating-tasks/Output/README.MD

```diff
@@ -0,0 +1,3 @@
+#### A screencast showing gulp watching and refreshing your browser.
+### Link to screencast
+[link to screen cast of gulp automation](http://youtu.be/zcBIFShQHNw?hd=1)
```

Comment:
I have done a re-screencast of the video showing gulp watching and refreshing on my browser